app/forms/state_file/nj_county_form.rb

```diff
@@ -0,0 +1,17 @@
+module StateFile
+  class NjCountyForm < QuestionsForm
+    set_attributes_for :intake,
+                       :county
+
+    validates :county, presence: true
+
+    def save
+      if @intake.county != self.county
+        @intake.municipality_code = nil
+        @intake.municipality_name = nil
```

Comment:
I'd like to write a test for this behavior (when the county is changed, the municipality code/name should get reset) but I'm not sure how to do so in the context of how the form specs are structured. How could I do that?